tests/ui/check-cfg/cargo-build-script.stderr

```diff
@@ -7,8 +7,8 @@ LL | #[cfg(has_foo)]
    = help: expected names are: `clippy`, `debug_assertions`, `doc`, `doctest`, `has_bar`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `ub_checks`, `unix`, and `windows`
    = help: consider using a Cargo feature instead
    = help: or consider adding in `Cargo.toml` the `check-cfg` lint config for the lint:
-            [lints.rust]
-            unexpected_cfgs = { level = "warn", check-cfg = ['cfg(has_foo)'] }
+   |        [lints.rust]
+   |        unexpected_cfgs = { level = "warn", check-cfg = ['cfg(has_foo)'] }
```

Comment:
it's a bit unfortunate that it breaks the simple copy/pasting of this suggestion[^1], but you can see that the suggestion is currently off-by-one because there wasn't any vertical line, so it's a loss for a win 

[^1]: suggestion which isn't (for the code) a real suggestion but a simple help text because you don't have the span for the `Cargo.toml`

Reply:
Yeah, I noticed this too as a potential issue. Just for now, I have reverted this change for the ascii art output, but enabling it again sometime in the future would be as simple as changing a single line.